GameDataParser/Parsers/SkillParser.cs

```diff
@@ -13,101 +14,127 @@ namespace GameDataParser.Parsers
 {
     public static class SkillParser
     {
-
         public static List<SkillMetadata> Parse(MemoryMappedFile m2dFile, IEnumerable<PackFileEntry> entries)
         {
             List<SkillMetadata> skillList = new List<SkillMetadata>();
-
-            foreach (PackFileEntry skill in entries)
+            foreach (PackFileEntry xmlFile in entries)
             {
-                if (!skill.Name.StartsWith("skill/10"))
-                    continue;
-
-                string skillId = Path.GetFileNameWithoutExtension(skill.Name);
-                SkillMetadata metadata = new SkillMetadata();
-                List<SkillMotion> motions = new List<SkillMotion>();
-                List<SkillLevel> skillLevel = new List<SkillLevel>();
-
-                metadata.SkillId = int.Parse(skillId);
-                Debug.Assert(metadata.SkillId > 0, $"Invalid Id {metadata.SkillId} from {skillId}");
-
-                //using XmlReader reader = m2dFile.GetReader(skill.FileHeader);
-                XmlDocument document = m2dFile.GetDocument(skill.FileHeader);
-
-                XmlNodeList basic = document.SelectNodes("/ms2/basic");
-                foreach (XmlNode node in basic)
+                // Parsing Skills
+                if (xmlFile.Name.StartsWith("skill"))
```

Comment:
Invert and use `continue`

Reply:
cannot be because it then won't get the other parse on 'table/job'

Reply:
Yes you can, it's called `||`...